packages/core/src/components/sections/ProductShelf/ProductShelf.tsx

```diff
@@ -72,6 +72,10 @@ function ProductShelf({
                   product={product.node}
                   index={idx + 1}
                   aspectRatio={aspectRatio}
+                  imgProps={{
+                    width: 216,
+                    height: 216,
```

Comment:
why 216? 👀

Reply:
that's the rendered size for this image using desktop
<img width="1968" alt="Screenshot 2023-04-28 at 10 45 40" src="https://user-images.githubusercontent.com/11325562/235165833-c54387d3-23ce-4283-964a-f21aabfc7c9b.png">